index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE HTML>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet" />
+</head>
+<body>
+    <div class="main-box">
+        <article id="temp-box">
```

Comment:
I would use a section or div tag here because you aren't displaying a bunch of text that would normally be in a article tag.